File: 프로그래머스/0/181836. 그림 확대/그림 확대.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(vector<string> picture, int k) {
    vector<string> answer(picture.size() * k);
    for(int i = 0; i < answer.size(); i++)
    {
        string temp = picture[i/k];
        string result = "";
        for(int j = 0; j < temp.size(); j++)
        {
            for(int m = 0 ; m < k; m++)
            {
                char ch = temp[j];
                result += ch;
            }
        }
        answer[i] = result;
    }
    return answer;
}

File: 프로그래머스/0/181836. 그림 확대/그림 확대.cs
using System;
using System.Linq;
public class Solution {
    public string[] solution(string[] picture, int k) {
        string[] answer = new string[picture.Length * k];
        for(int i = 0; i < answer.Length; i++)
        {
            string temp = picture[i/k];
            string result = "";
            for(int j = 0; j < temp.Length; j++)
            {
                for(int m = 0; m < k; m++)
                {
                    char ch = temp[j];
                    result += ch;
                }
            }
            answer[i] = result;
        }
        return answer;
    }
}